app/components/createService/createservice.html

```diff
@@ -445,6 +446,52 @@
           <!-- tab-placement -->
           <div class="tab-pane" id="placement" ng-include="'app/components/createService/includes/placement.html'"></div>
           <!-- !tab-placement -->
+          <!-- tab-resources -->
+          <div class="tab-pane" id="resources">
+            <form class="form-horizontal" style="margin-top: 15px;">
+              <div class="form-group">
+                <!-- cpu-limit-input -->
+                <label for="cpu-limit" class="col-sm-2 col-lg-2 control-label text-left">CPU limit<portainer-tooltip position="top" message="Maximum number of CPU the service can use (0 to unlimited)."></portainer-tooltip></label>
```

Comment:
Please replace the tooltip message with: `Maximum CPU usage per task (set to 0 for unlimited)`